actor/builtin/builtin.go

```diff
@@ -12,16 +14,58 @@ import (
 	cid "github.com/ipfs/go-cid"
 )
 
-// Actors is list of all actors that ship with Filecoin.
-// They are indexed by their CID.
-var Actors = map[cid.Cid]exec.ExecutableActor{}
+type codeVersion struct {
+	code    cid.Cid
+	version uint64
+}
+
+type Actors struct {
+	actors map[codeVersion]exec.ExecutableActor
+}
+
+func (ba Actors) GetBuiltinActorCode(code cid.Cid, version uint64) (exec.ExecutableActor, error) {
```

Comment:
Since the package is `builtin`, remove this from the type and function names. e.g. `builtin.GetActorCode`